update for python scripts so no python2.7 is needed

BehleZebub opened the pull request:
As explained in [Issue 736](https://github.com/eried/portapack-mayhem/issues/736) there are python scripts which require python2.7.

These changes will remove this requirement and update all scripts to use python3 and use correct python3 syntax.

GullCode commented:
This is already the case.
You need to update your copy of next. Libopencm3 (part of hackrf submodule) is now using python3 as a defaut. The Debian ARM wiki page is also updated. Notice part 6.

BehleZebub commented:
If you compile firmeware on a non unix machine the shebang line will be ignored and the default python interpreter will be used.
Therefore a non unix user compiling firmware may not notice any change.

But for unix machines I changed the shebang line to use the correct version of python interpreter.
This applies to:
- firmware/tools/extract_cpld_data.py
- firmware/tools/make_baseband_file.py
- firmware/tools/make_bitmap.py
- firmware/tools/make_dcs.py
- firmware/tools/make_image_chunk.py
- firmware/tools/make_spi_image.py
- firmware/tools/make_image_chunk.py

I also changed print() to have the minimum required brackets in all the above where print() was used and additionally in
- firmware/tools/fir_lpf.py

Now thinking about it maybe fir_lpf.py should also have a shebang line for consistency...

GullCode commented:
I'll test this and merge if it's working.
Please add the shebang you talked about.

BehleZebub commented:
Here it is.
I tried to compile the firmware (both original fork and then with my changes), compared both bin files and there were some minor differences.
I guess that is because of string formatting (might be a result from python2.7 vs python3 differences) and version numbering.

My portapack runs fine with both firmwares, but I did not test everything thoroughly so please double check it. 
Even if I'm pretty sure, it won't have negative side effects.

GullCode commented:
Compiled and tested, looks fine here too. Anyway if there is a problem we will revert / search why.

GullCode reviewed (commented):
Everything looks fine
